### PR TITLE
Cb 2

### PR DESCRIPTION
**Pull request by asi345**
The overall body of the project is ready, anyone can pull and add their part of the API code after reviewing for possible mistakes.

### CODE REVIEW

**Comment by kocasariumut** [comment]
It is working as intended. Approved.

File: practice-app/profile/profile.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+'''
+
+@author Ismail Ata Inan
+
+'''
```
Comment (CodeBlo) [inline practice-app/profile/profile.py:5]: The author lines can be deleted.

**Review by kocasariumut — APPROVED** [review]
It is working as intended. Approved.

File: practice-app/app.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+'''
+
+@author Ismail Ata Inan
+
+'''
```
Comment (CodeBlo) [inline practice-app/app.py:5]: The author lines can be deleted.

**Review by CodeBlo — COMMENTED** [review]
Could delete the author but overall is fine.

File: practice-app/database.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+'''
+
+@author Ismail Ata Inan
+
+'''
```
Comment (CodeBlo) [inline practice-app/database.py]: Same here :)

File: practice-app/profile/profile.py (hunk 1 above)
Comment (onurcanavci) [reply practice-app/profile/profile.py:5]: I agree to Abdulkadir

**Review by onurcanavci — APPROVED** [review]

**Review by CodeBlo — APPROVED** [review]

**Review by onurcanavci — APPROVED** [review]

File: practice-app/profile/profile.py (hunk 1 above)
Comment (asi345) [reply practice-app/profile/profile.py]: I removed it

File: practice-app/app.py (hunk 2 above)
Comment (asi345) [reply practice-app/app.py]: I removed it

File: practice-app/database.py (hunk 3 above)
Comment (asi345) [reply practice-app/database.py]: I removed it

Comment (CodeBlo) [reply practice-app/database.py]: I think it is bugged. I made this comment before the removal :(

**Pull request merged**